src/NzbDrone.Core/DataAugmentation/Scene/SceneMappingService.cs

```diff
@@ -154,7 +154,33 @@ private SceneMapping FindTvdbId(string title)
                 RefreshCache();
             }
 
-            return _getTvdbIdCache.Find(title.CleanSeriesTitle());
+            var candidates = _getTvdbIdCache.Find(title.CleanSeriesTitle());
+
+            if (candidates == null)
+            {
+                return null;
+            }
+
+            if (candidates.Count == 1)
+            {
+                return candidates.First();
+            }
+
+            var exactMatch = candidates
```

Comment:
Only thing was a nitpick here to move the orderby to the first line then align on the `.` (Pedantic I know).